scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -64,7 +64,6 @@ core.domain.rules_registry_test
 core.domain.search_services_test
 core.domain.skill_domain_test
 core.domain.skill_services_test
-core.domain.state_domain_test
```

Comment:
We can't remove the file from this list until we get 100% coverage. Right now, the coverage checks are failing: https://github.com/oppia/oppia/actions/runs/3665331064/jobs/6196331027#step:8:1003. You can either:

1. Write more tests to get this file to full coverage and complete the task in #14219 
2. Merge this PR without getting to 100%. I'm okay with this since more tests are better, but since it doesn't complete the task, it won't count as one of the 2 PRs needed to complete onboarding